daemon/src/maker_cfd_actor.rs

```diff
@@ -27,7 +30,10 @@ pub enum Command {
         id: TakerId,
     },
     IncProtocolMsg(SetupMsg),
-    CfdSetupCompleted(FinalizedCfd),
+    CfdSetupCompleted {
+        order_id: OrderId,
+        dlc: FinalizedCfd,
```

Comment:
Why `dlc`?

Reply:
Because it is the parameters needed for the DLC. I think that is what it is. I can also name it `finalized_cfd`, but I found that name a bit arbitrary.

Reply:
Yeah that name is totally up for change haha. It was just the first thing that come to my mind when we prototyped the protocol integration :)

Reply:
Naming it `Dlc` sounds good to me!